tooling/nargo_fmt/src/utils.rs

```diff
@@ -146,9 +146,12 @@ impl HasItem for Param {
     fn format(self, visitor: &FmtVisitor, shape: Shape) -> String {
         let pattern = visitor.slice(self.pattern.span());
         let visibility = match self.visibility {
-            Visibility::Public => "pub",
-            Visibility::Private => "",
-            Visibility::DataBus => "call_data",
+            Visibility::Public => "pub".to_string(),
+            Visibility::Private => "".to_string(),
+            Visibility::DataBus(x) => {
+                let y = if x > 0 { x - 1 } else { 0 };
```

Comment:
This appears to hide the return data case?

Reply:
No because +1 was added only in the call data case, but anyway I changed to a return data visibility as you suggested.